File: src/main/java/no/nav/syfo/model/Arbeidsforhold.java
package no.nav.syfo.model;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDate;
import java.util.List;

@Data
@Accessors(fluent = true)
public class Arbeidsforhold {
    public String orgnummer;
    public List<Stilling> stillinger;
    public LocalDate fom;
    public LocalDate tom;
}

File: src/main/kotlin/no/nav/syfo/api/v3/domain/Arbeidsforhold.kt
package no.nav.syfo.api.v3.domain

import no.nav.syfo.model.Stilling
import java.math.BigDecimal

data class Arbeidsforhold (
    val yrke: String,
    val prosent: BigDecimal
)

fun Stilling.mapToArbeidsforhold(): Arbeidsforhold {
    return Arbeidsforhold(
        yrke = this.yrke,
        prosent = this.prosent
    )
}
